projects/kit/components/input-date-multi/input-date-multi.module.ts

```diff
@@ -0,0 +1,49 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {MaskitoModule} from '@maskito/angular';
+import {TuiLetModule, TuiMapperPipeModule, TuiPreventDefaultModule} from '@taiga-ui/cdk';
+import {
+    TuiCalendarModule,
+    TuiHostedDropdownModule,
+    TuiLinkModule,
+    TuiPrimitiveTextfieldModule,
+    TuiSvgModule,
+    TuiTextfieldComponent,
+    TuiTextfieldControllerModule,
+    TuiWrapperModule,
+} from '@taiga-ui/core';
+import {TuiInputTagModule} from '@taiga-ui/kit/components/input-tag';
+import {TuiValueAccessorModule} from '@taiga-ui/kit/directives';
+import {PolymorpheusModule} from '@tinkoff/ng-polymorpheus';
+
+import {TuiInputDateMultiComponent} from './input-date-multi.component';
+import {TuiInputDateMultiDirective} from './input-date-multi.directive';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        MaskitoModule,
+        PolymorpheusModule,
+        TuiWrapperModule,
+        TuiPreventDefaultModule,
```

Comment:
This is deprecated, let's use `(mousedown.silent.prevent)="0"` instead. Also make sure all modules are used, please. I think `TuiPrimitiveTextfieldModule` is not, maybe some other modules too.

Reply:
fixed